RELEASE/scripts/autoscend/quests/level_05.ash

```diff
@@ -1,19 +1,40 @@
-boolean L5_getEncryptionKey()
+//L5 quest progress notes:
+//unstarted
+//started == acquired [Cobb's Knob map] from council
+//step1 == used [Cobb's Knob map] with encryption key to unlock insides.
+//finished == killed the king. note that you still need to visit council after the question is finished to get reward.
+
+boolean L5_waitForSemirare()
```

Comment:
Can we reserve `LX_something()` for functions that might spend adventures, use items, or otherwise modify quest state? would prefer if this was labelled more clearly

Reply:
I initially reused it in a different location before removing the reuse and then I forgot to undo the spinoff into a different function. I renamed it to `L5_shouldWaitForSemirareL5()` ... would you prefer that i remove it entirely and return to the prior way of doing things on this?

Reply:
Alright. went ahead and removed `L5_shouldWaitForSemirareL5()` which significantly trimmed down this commit's size. please take another look when you have the time.